ballet/util/__init__.py

```diff
@@ -1,7 +1,12 @@
+from contextlib import redirect_stderr, redirect_stdout
```

Comment:
Since contextlib does not support `redirect_stderr` in Python 3.4, [here](https://stackoverflow.com/questions/6735917/redirecting-stdout-to-nothing-in-python) is an alternative. Thoughts?